### PR TITLE
resources: Add fixture definition for Bright XBAR

### PR DESCRIPTION
**Pull request by cybic**
Added fixture definition for the Bright XBAR 12 cell RGBAWUV LED bar

### CODE REVIEW

**Comment by mcallegari** [comment]
Please name modes like "96 Channel"
Please add color macros

**Comment by cybic** [comment]
Thanks for the feedback! I've tried to fix it ;)

> Please name modes like "96 Channel"
Fixed

> Add color macros
Fixed

> Add shutter presets
Fixed as best I could. Didn't find anything that matched "Strobe Audio Slow -> Fast".

> Also doesn't pass fixture validator https://www.qlcplus.org/fixture_validator.php
I actually hadn't seen the validator. I probably should have read some documentation.  Thank you for the patience :)

File: resources/fixtures/FixturesMap.xml [hunk 1]
```diff
@@ -280,6 +280,9 @@
     <F n="Botex-DSP-405" m="DSP-405"/>
     <F n="Botex-SP-1500" m="SP-1500"/>
   </M>
+  <M n="Bright">
+    <F n="Bright-XBAR" m="Bright XBAR"/>
```
Comment (mcallegari) [inline resources/fixtures/FixturesMap.xml:284]: This is wrong. Manufacturer should be "Bright".
Please remove this change and leave this to me. (must be created with the dedicated Python script)

Comment (cybic) [reply resources/fixtures/FixturesMap.xml]: Ah, I saw the addition of the FixtureMap entry in some other commit adding a fixture, but that was of course your commit.
Done!

**Comment by mcallegari** [comment]
Thanks :+1:

**Pull request merged**